nicegui/elements/qtable.js

```diff
@@ -0,0 +1,75 @@
+function eval_fields(cols) {
+    // For each Field, Sort and Format field within the column definition,
+    // look for '=>' and if so, try to eval to run the function.
+
+    cols.forEach((col) => {
+        if (!col.field || !col.field.includes("=>")) return;
+        col.field = eval(col.field);
+
+        if (!col.sort || !col.sort.includes("=>")) return;
+        col.sort = eval(col.sort);
+
+        if (!col.format || !col.format.includes("=>")) return;
+        col.format = eval(col.format);
+    });
+}
+
+export default {
+    template: `
+        <q-table
+            :rows="rows"
+            :columns="columns"
+
+            :row-key="this.selection_key"
+            :selection="this.selection_mode"
+
+            v-model:selected="selected"
+            @selection="onSelection"
+
+            :filter="filter"
+        >
+
+        <template v-slot:top-right>
```

Comment:
I would suggest to add a v-if on that slot here : unless I am wrong, it means here that the slot is "mandatory" as I can't go without the search bar here.